frontend/packages/dev-console/src/components/svg/SvgDropShadowFilter.tsx

```diff
@@ -19,24 +19,48 @@ const SvgDropShadowFilter: React.FC<SvgDropShadowFilterProps> = ({
   stdDeviation = 2,
   floodColor = '#030303',
   floodOpacity = 0.2,
-}) => (
-  <SVGDefs id={id}>
-    <filter
-      id={id}
-      x={`-${stdDeviation * 12.5}%`}
-      y={`-${stdDeviation * 12.5}%`}
-      width={`${100 + stdDeviation * 25}%`}
-      height={`${100 + stdDeviation * 25}%`}
-    >
-      <feDropShadow
-        dx={dx}
-        dy={dy}
-        stdDeviation={stdDeviation}
-        floodColor={floodColor}
-        floodOpacity={floodOpacity}
-      />
-    </filter>
-  </SVGDefs>
-);
+}) => {
+  if (window.navigator.userAgent.includes('Edge')) {
+    // feDropShadow is not supported by Edge
```

Comment:
I specifically wrote this as a whole if-block by itself as Edge is going Chromium in the new year, perhaps we just drop this whole if when that day comes.